torch/_inductor/codegen/cpp.py

```diff
@@ -1326,6 +1336,10 @@ def to_dtype(x, dtype, src_dtype=None):
         ], f"{__name__} does not support {dtype}"
         node: torch.fx.Node = V.interpreter.current_node
         assert node and isinstance(node, torch.fx.Node)
+        if node.target == "store":
+            assert dtype == torch.float
+            assert src_dtype in DTYPE_LOWP_FP
+            return f"cvt_lowp_fp_to_fp32<{DTYPE_TO_CPP[src_dtype]}>({x})"
```

Comment:
This looks a bit hacky. Perhaps you can factor out a method to return the line for such a conversion, used by both `to_dtype` and `store`?

Reply:
Removed this part in to_dtypes.
Instead, just add a function to generate `to_fp32_expr`